astropy/utils/iers/iers.py

```diff
@@ -790,7 +790,11 @@ def open(cls):
 
         """
         if not conf.auto_download:
-            cls.iers_table = cls.read()
+            # If auto_download is changed to False mid-session, iers_table may have already been
+            # made from non-bundled files, so it should be remade from bundled files
+            if not getattr(cls, "_iers_table_bundled", None):
```

Comment:
Unless I'm missing something this should be equivalent ?
```suggestion
            if not hasattr(cls, "_iers_table_bundled"):
```

Reply:
Yup, that works, changed